examples/edit_box.rb

```diff
@@ -0,0 +1,12 @@
+require "scarpe"
+
+Scarpe.app do
+  para "Manuscript:"
+  @manuscript = edit_box(width: "100%") do
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim."
+  end
+  @char_count = para "#{@manuscript.text.length} characters"
+  @manuscript.change { |text|
```

Comment:
I wasn't sure if it was better to have the example use the value passed to the block or if it should be
```ruby
@manuscript.change {
  @char_count.replace("#{@manuscript.text.length} characters")
}
```
instead